src/combobox/ComboBox.ts

```diff
@@ -394,7 +396,7 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			v('button', {
 				'aria-controls': menuId,
 				bind: this,
-				classes: this.classes(css.arrow),
+				classes: this.classes(css.trigger, iconCss.icon, iconCss.chevronDown),
```

Comment:
Should we include such specific class names within the widget code? This could get semantically awkward if someone wants to style the button with a "+"-icon or something. I'm not sure of the best way to get around it, but my first thought is to leave `iconCss` out of `ComboBox` and handle it within the css modules using variables and imports.